apps/server/src/shared/domain/entity/school.entity.ts

```diff
@@ -84,15 +88,36 @@ export class School extends BaseEntity {
 
 	constructor(props: ISchoolProperties) {
 		super();
-		if (props.externalId) this.externalId = props.externalId;
-		if (props.inMaintenanceSince) this.inMaintenanceSince = props.inMaintenanceSince;
-		if (props.inUserMigration !== null) this.inUserMigration = props.inUserMigration;
-		if (props.oauthMigrationPossible !== null) this.oauthMigrationPossible = props.oauthMigrationPossible;
-		if (props.oauthMigrationMandatory !== null) this.oauthMigrationMandatory = props.oauthMigrationMandatory;
+		if (props.externalId) {
+			this.externalId = props.externalId;
+		}
+		if (props.inMaintenanceSince) {
+			this.inMaintenanceSince = props.inMaintenanceSince;
+		}
+		if (props.inUserMigration) {
+			this.inUserMigration = props.inUserMigration;
```

Comment:
why changing the condition !== null? 
I need to check if this was changed recently, but there was some logic where inUserMigration is set to false on purpose

Reply:
because this checks on null, undefined and all other null-like states. so it's more safe than before